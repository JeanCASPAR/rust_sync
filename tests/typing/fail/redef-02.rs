use rust_sync::sync;

sync! {
    #![pass(1)]

    node oui() = (hello)
    where
        hello : int = hello,
        hello : int = hello;
}

fn main() {}
