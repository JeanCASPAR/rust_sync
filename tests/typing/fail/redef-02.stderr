error: variable `hello` is defined twice
 --> tests/typing/fail/redef-02.rs:9:9
  |
9 |         hello : int = hello;
  |         ^^^^^
  |
help: the variable was previously defined here
 --> tests/typing/fail/redef-02.rs:8:9
  |
8 |         hello : int = hello,
  |         ^^^^^
